docs/en/sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md

```diff
@@ -210,6 +213,30 @@ If `property.group.id` is not specified, StarRocks generates a random value base
     "property.sasl.password" = "xxxxxx"
     ```
 
+  - Access Kafka using the SASL_PLAINTEXT security protocol and SASL/GSSAPI (Kerberos) authentication mechanism:
```

Comment:
align the code comments with original code examples
![image](https://github.com/StarRocks/starrocks/assets/98087056/8d8b56af-b4d0-4fd5-8180-90a9171561cc)